project/Settings.scala

```diff
@@ -26,7 +26,10 @@ object Settings {
     Test / console / scalacOptions --= Seq("-Ywarn-unused:imports", "-Xfatal-warnings"), //disable unused fatal warning in `sbt test:console`
     Test / scalacOptions --= List("-Ywarn-dead-code", "-deprecation", "-Xfatal-warnings"),
     addCompilerPlugin("com.olegpy" %% "better-monadic-for" % "0.3.1"),
-    addCompilerPlugin(scalafixSemanticdb)
+    // Hard-coding the semanticdbVersion to make it work with Scala 2.13.6. It's possible this may be fixed
+    // in a future scalafix version. See: https://scalacenter.github.io/scalafix/docs/users/installation.html
+    semanticdbEnabled := true,
```

Comment:
This looks like the right way to install `scalafix` for sbt 1.3 based on https://scalacenter.github.io/scalafix/docs/users/installation.html. 

Although I had to hard-code the version to work with Scala 2.13.6: https://mvnrepository.com/artifact/org.scalameta/semanticdb-scalac